### PR TITLE
ARROW-10777: [Packaging][Python] Build sdist by Crossbow

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kou** [comment]
@github-actions crossbow submit sdist

File: ci/docker/python-sdist.dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+RUN apt-get update -y -q && \
+    apt-get install -y -q --no-install-recommends \
+        git \
+        python3-pip && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists*
+
+COPY python/requirements-build.txt \
+     /arrow/python/requirements-build.txt
```
Comment (xhochy) [inline ci/docker/python-sdist.dockerfile:33]: Ubuntu 20.04 has setuptools 44, nevertheless I made https://github.com/apache/arrow/pull/9153 to ensure that we have a recent enough setuptools version to support markdown.

**Review by xhochy — COMMENTED** [review]
Looks good, leaving open for @kszucs

**Comment by kszucs** [comment]
Added a packaging group which includes the sdist task.

**Review by kszucs — APPROVED** [review]
Awesome, thanks @kou!

**Comment by kou** [comment]
@github-actions crossbow submit -g packaging

**Comment by kou** [comment]
+1

**Pull request closed without merging**